include/mbgl/util/filesystem.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <ghc/filesystem.hpp>
```

Comment:
Why is this inside `include/mbgl/util`? It is not used by core.

Reply:
This looks like something we'll eventually take in use in core. But I'm :+1: in putting this inside `render-test` for now.